packages/pill/index.js

```diff
@@ -0,0 +1,104 @@
+import "@warp-ds/icons/elements/close-16";
+import { activateI18n } from "../i18n";
+import { classNames } from "@chbphone55/classnames";
+import { html } from "lit";
+import { i18n } from "@lingui/core";
+import { messages as enMessages } from "./locales/en/messages.mjs";
+import { messages as fiMessages } from "./locales/fi/messages.mjs";
+import { messages as nbMessages } from "./locales/nb/messages.mjs";
+import { pill as ccPill } from "@warp-ds/css/component-classes";
+import WarpElement from "@warp-ds/elements-core";
+
+class WarpPill extends WarpElement {
+  static styles = [WarpElement.styles];
+
+  static properties = {
+    canClose: { type: Boolean },
+    suggestion: { type: Boolean },
+    openSRLabel: { type: String },
+    closeSRLabel: { type: String },
+  };
+
+  constructor() {
+    super();
+    activateI18n(enMessages, nbMessages, fiMessages);
+    this.canClose = false;
+    this.suggestion = false;
+
+    this.openFilterSrText = i18n._({
+      id: "pill.aria.openFilter",
+      message: "Open filter",
+      comment: "Fallback screenreader message for open filter",
+    });
+
+    this.removeFilterSrText = i18n._({
+      id: "pill.aria.removeFilter",
+      message: "Remove filter",
+      comment: "Fallback screenreader message for removal of the filter",
+    });
+  }
+
+  get _labelClasses() {
+    return classNames({
+      [ccPill.button]: true,
+      [this.suggestion ? ccPill.suggestion : ccPill.filter]: true,
+      [ccPill.label]: true,
+      [this.canClose ? ccPill.labelWithClose : ccPill.labelWithoutClose]: true,
+    });
+  }
+
+  get _closeClasses() {
+    return classNames({
+      [ccPill.button]: true,
+      [this.suggestion ? ccPill.suggestion : ccPill.filter]: true,
+      [ccPill.close]: true,
+    });
+  }
+
+  _onClick() {
+    this.dispatchEvent(new CustomEvent("w-pill-click", { bubbles: true, composed: true }));
```

Comment:
These were missing. I looked around other Design Systems and they seem to be prepending their events with something, so I went with `w-{componentName}`. Any opinions @BalbinaK @felicia-haggqvist @magnuh @Skadefryd ?

Reply:
Sounds good to me. Shoelace library seems to do it this way and they added a section on custom events to the contributing file, which explains their choice: https://github.com/shoelace-style/shoelace/blob/83fe1ff28e88b1982b528161bf8215d61b4f4592/docs/pages/resources/contributing.md?plain=1#L339C1-L343C313

Does it mean we would use the pill component for example like so?
```
<w-pill @w-pill-click="handleClick">...</w-pill>
```